system_test/only_local/aest_heavy_sync_SUITE_data/epoch.yaml.mustache

```diff
@@ -31,7 +31,7 @@ mining:
     autostart: {{mining.autostart}}
     cuckoo:
         miner:
-            executable: mean28s-generic
-            extra_args: ""
-            node_bits: 28
+            executable: mean29-generic
+            extra_args: "-t 5"
+            edge_bits: 29
```

Comment:
I see there is no `mean27`. Running five threads means test won't be too much slower?